<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>                                                        Вопрос</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mulish,Arial,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mulish,Arial,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>87</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
